sdk/attestation/azure-security-attestation/tests/test_azure_attestation.py

```diff
@@ -64,209 +57,151 @@ class AzureAttestationTest(AzureTestCase):
     def setUp(self):
             super(AzureAttestationTest, self).setUp()
 
-    @pytest.mark.live_test_only
-    def test_shared_getopenidmetadata(self):
-        attest_client = self.shared_client()
-        open_id_metadata = attest_client.metadata_configuration.get()
+    @AttestationPreparer()
+    def test_shared_getopenidmetadata(self, attestation_location_short_name):
+        attest_client = self.shared_client(attestation_location_short_name)
+        open_id_metadata = attest_client.get_openidmetadata()
```

Comment:
I looked at the call for `get_openidmetadata`, we always remove this request from the recording to avoid leaks, any tests that run this call will have to be marked with the `@pytest.mark.live_test_only`, in the meantime we can discuss removing this with  the Python crew.